index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <div id="navbar">
```

Comment:
How about using a <nav> element for your navigation bar instead of <div>. Using [semantic elements](http://diveintohtml5.it/semantics.html) makes your code more accessible, easier to read, and more professional. What could a future employee ask for more?